rules/assertion-arguments.js

```diff
@@ -137,14 +137,20 @@ module.exports = context => {
 	});
 };
 
-module.exports.schema = [{
-	type: 'object',
-	properties: {
-		message: {
-			enum: [
-				'always',
-				'never'
-			]
-		}
+module.exports = {
+	create,
+	meta: {
+		docs: {},
```

Comment:
I'll take a look, but this is where core rules have their `recommended` setting from what I can see, so it might be a good spot to put that value. If we go that, there's no value in removing it.